### PR TITLE
[Doc] Fixed typos

### PR DESCRIPTION
**Pull request by pborreli**

### CODE REVIEW

File: docs/Tutorial.md [hunk 1]
```diff
@@ -222,7 +222,7 @@ We want to send off the request after the component has finished loading. `compo
   },
 ```
 
-Now add `fetchData` function used above to our main component. This method will be respondible for handling data fetching. All you need to do is call `this.setState({movies: data})` after resolving the promise chain because the way React works is that `setState` actually triggers a re-render and then the render function will notice that `this.state.movies` is no longer `null`.  Note that we call `done()` at the end of the promise chain - always make sure to call `done()` or any errors thrown will get swallowed.
```
Comment (pborreli) [inline docs/Tutorial.md]: respondible -> responsible (the GitHub diff editor doesn't show it)

Comment (sahrens) [inline docs/Tutorial.md]: This file was cleaned up in some other PRs, but your other changes are still valid - can you rebase?

Comment (pborreli) [inline docs/Tutorial.md]: done

**Comment by pborreli** [comment]
rebased

**Comment by brentvatne** [comment]
Looks like this has been fixed, thanks!

**Comment by pborreli** [comment]
yes indeed, it has been fixed by #850 
This policy of merging is wrong, it's not the first time.
Had to rebase twice because other PR were merged while mine was not.
Thanks :pray:

**Pull request closed without merging**